File: Data Structures & Algorithms/longest-consecutive-sequence/submission-19.java
class Solution 
{
    public int longestConsecutive(int[] nums) 
    {
        HashSet<Integer> set = new HashSet<Integer>();

        for(int i = 0; i < nums.length; i++)
        {
            set.add(nums[i]);
        }

        int longest = 0;

        for(int num : set)
        {
            int longu = 0;
            if(!set.contains(num - 1))
            {
                int length = 1;
                while(set.contains(num + length))
                {
                    length++;
                }

                longu = length;
            }

            longest = Math.max(longest, longu);
        }
        return longest;
    }
}

File: Data Structures & Algorithms/longest-substring-without-duplicates/submission-19.py
class Solution:
    def lengthOfLongestSubstring(self, s: str) -> int:
        best = 0

        for i in range(len(s)):
            for j in range(i, len(s)):
                if len(s[i:j+1]) != len(set(s[i:j+1])):
                    break
                else:
                    best = max(best, len(s[i:j+1]))

        return best